packages/conf-openssl/conf-openssl.1/files/osx-build.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 
-if [ -d "/usr/local/opt/openssl/" ]; then
+if [ -d "$HOME/.nix-profile/lib/pkgconfig/" ]; then
+  # Nix on macOS
+  PKG_CONFIG_PATH=$HOME/.nix-profile/lib/pkgconfig 
```

Comment:
Sorry, I just noticed this: I think you need to add ` pkg-config openssl` to this line

Reply:
oh wow my bad, nice catch. Will do right now

Reply:
Done and updated the hash too